src/components/wallet-box.tsx

```diff
@@ -63,6 +52,25 @@ export function WalletBox(): JSX.Element {
     logout();
   }
 
+  function blankedAddress(): string {
+    return `${address?.slice(0, 6)}...${address?.slice(address?.length - 5)}`;
+  }
```

Comment:
It could make sence to extract such functionality to an Utility class as a static function.